float32 xMin
float32 yMin
float32 xMax
float32 yMax

